docker-compose.yml

```
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+
+  postgis:
+    image: mdillon/postgis:9.6

```

```suggestion
    image: kartoza/postgis:11.0-2.5

```

Does that work too? Let's go a bit more modern ;)